server/graphql-schema.js

```diff
@@ -393,8 +393,11 @@ const graphqlSchema = gql`
         """
         Since each TestPlan originates from a CSV, this number corresponds to
         the row within the CSV where this test originated.
+
+        Float type because presentationNumber fields from the source *.csv
```

Comment:
For reviewers, I cannot at this moment say with certainty why this didn't affect previous v2 test format imports that also use decimal numbers the same way. I'll continue to investigate why through the commit which caused it: https://github.com/w3c/aria-at/commit/af80e48dc9156f2749b2ecb83d93826d6bf07e6e.